### DISCUSSION TITLE
Real-Time Streaming Hackathon in July 2023

### DISCUSSION THREAD

**Discussion by asharamseervi** [ant-media/Ant-Media-Server#5326] [Announcements]
👋 Greetings, community!
Are you ready for an adventure?

We are thrilled to announce the Ant Media "**Real-Time Streaming Hackathon**" for **July 2023**!

## The Challenge
Are you ready to revolutionize the world of streaming? Unleash your creativity and expertise in real-time streaming technologies. Whether you're a seasoned developer or a passionate newcomer, this hackathon welcomes participants of all skill levels.

- Build a real-time streaming application that uses Ant Media Server.
- The application can be anything you want, as long as it uses Ant Media Server to stream live or on-demand video. Hackathons are for fun, but if you could make it something usable that users can adopt, your project will likely get extra points.
- You can use any programming language or framework you want.
- 
Starter kit: https://drive.google.com/file/d/1OfkxEE6CSKOZ5CjmuGG2T27nLr6yijWG/view?usp=drive_link
Sample ideas: https://drive.google.com/file/d/1D6cIssho1_oQAz-wekkkvHta7juxil6r/view?usp=drive_link

Don't forget to [Submit your Ideas](https://docs.google.com/forms/d/e/1FAIpQLSdTSzOr7c9jRM3KC2WJTnftunau7BQPBfWDtEwCxMcmqM7Jyg/viewform?usp=pp_url)

## The prize pool:
To make things even more exciting, we have put together a generous prize pool to reward your hard work and innovative solutions. 

With a total prize pool of $6000, you'll have the opportunity to compete for incredible rewards and recognition within the streaming community.

🥇 1st prize:  $3000
🥈 2nd prize: $2000
🥉 3rd prize:  $1000

## The Dates and Timeline
Mark your calendars! The Real-Time Streaming Hackathon will take place from **22nd July** to **30th July**.

### Here's a breakdown of the key dates 🗓:
- 03rd July 2023: Hackathon Announcement
- 05th July 2023: Registration starts at [https://antmedia.io/hackathon/](https://antmedia.io/hackathon/)
- 22nd July 2023: Pre-event - 01:00 PM CET // https://meet.google.com/dxs-vzfk-gkk
- 29th July 2023: Hackathon Day 1 - Q&A with Mentors - 10:00 AM CET
- 30th July 2023: Hackathon Day 2 - Final Video Presentations & Award Ceremony - 03:00 PM CET

We can't wait to see the innovative solutions you'll come up with. 

💡 Remember, the hackathon is not just about winning; this is an exciting opportunity for all of you to expand your knowledge, demonstrate your expertise, build connections and collaboration and make valuable contributions to our vibrant community. Let's code, learn, and have some fun!

📢 So, what are you waiting for? Spread the word, and gather your coding comrades! Join us in the Real-Time Streaming Hackathon and demonstrate your skills and passion for streaming technology. Together, we will ignite a streaming revolution shaping the future of real-time streaming!

Happy hacking and streaming! 🚀

_PS: Stay tuned! We'll be sharing more exciting details soon. For easy access, consider bookmarking this thread._

**Comment by umtcylmz** [comment]
Dear community 😊,

We’re delighted to announce that Real-Time Streaming Hackathon ended successfully. Congratulations to all that participated in this journey 👏🏻👏🏻💫

The Winners are:

🥇1st Place: Team AGR HUB - CamOS by [Tan Do](https://www.linkedin.com/in/ACoAABTWlyUBOMbYXDXNQRUue34wcqehJuwH62g)
🥈2nd Place: Balazs Jantek - HTML Overlays by [Balázs Jantek](https://www.linkedin.com/in/ACoAAAtL93sBalNHKuPtmUun3myZbSp6t5BUykw)
🥉3rd Place: Team Rixel - Remote Trader by [Richárd Ádám Dr. Vécsey](https://www.linkedin.com/in/ACoAACm7Jo0B0S-1dKjN2as3-PSdbW5w3jpd9MQ) and [Axel Ország-Krisz Dr.](https://www.linkedin.com/in/ACoAAChkMYcBcy3btjoAaHkN_uUgXbT-YCUqeCk)
🏅Jury Special Award: Rahul Kumar - WordPress Plugin for AMS by [Rahul kumar](https://www.linkedin.com/in/ACoAACmE9fwBxluhllG9MFxYg_MHrAOJrvTBiAg)

Check all the presentations and share your thoughts with us in the comments. 😊👇
https://lnkd.in/dVCHB_5k
